qa/automation/Starynina.Scenarios/3. Open job positions page/3.2.Filter of open job positions.feature

```diff
@@ -0,0 +1,71 @@
+Feature: 3.2. List of open job positions
+
```

Comment:
I think you need to merge files 3.1 and 3.2, they have the same name in your file “List of open job positions”

after merging, name the file a little differently, for example : filter search check, since just the open page without filters is also the result of an empty search